src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingNamedTypeSymbol.cs

```diff
@@ -122,17 +122,37 @@ public override IEnumerable<string> MemberNames
 
         public override ImmutableArray<Symbol> GetMembers()
         {
-            return this.RetargetingTranslator.Retarget(_underlyingType.GetMembers());
+            var result = this.RetargetingTranslator.Retarget(_underlyingType.GetMembers());
+            if (!this.IsTupleType)
+            {
+                return result;
+            }
+
+            // For ValueTuple definitions, we may have to re-insert tuple error fields
+            return this.AddOrWrapTupleMembers(result).ToImmutableAndFree();
```

Comment:
>AddOrWrapTupleMembers [](start = 24, length = 21)

Symbols for real fields are going to be cached and reused by RetargetingTranslator. I think we should find a way to reuse symbols for added tuple error fields in a similar fashion.